Makefile.config

```diff
@@ -54,6 +54,18 @@ else
         $(warning Warn: Building without nftables support)
 endif
 
+# -Wdangling-pointer results in false warning when we add a list element to
+# local list head variable. It is false positive because before leaving the
+# function we always check that local list head variable is empty, thus
+# insuring that pointer to it is not dangling anywhere, but gcc can't
+# understand it.
+# Note: There is similar problem with kernel list, where this warning is also
+# disabled: https://github.com/torvalds/linux/commit/49beadbd47c2
+ifeq ($(call try-cc,$(FEATURE_TEST_NO_DANGLING_POINTER),-Wno-dangling-pointer),true)
+        $(info Note: Building criu with -Wno-dangling-pointer.)
+        FEATURE_DEFINES += -Wno-dangling-pointer
```

Comment:
I don't like the idea to use FEATURE_DEFINES for  anything else except defines. Here we can avoid try-cc by using -Wno-unknown-warning-option.

Reply:
Ok, I've updated PR accordingly.